src/Workleap.OpenApi.MSBuild/HttpClientWrapper.cs

```diff
@@ -41,7 +41,7 @@ public async Task DownloadFileToDestinationAsync(string url, string destination,
 
     private static async Task SaveFileFromResponseAsync(string destination, Stream responseStream, CancellationToken cancellationToken)
     {
-        using var fileTarget = new FileStream(destination, FileMode.Create, FileAccess.Write, FileShare.None);
+        await using var fileTarget = new FileStream(destination, FileMode.Create, FileAccess.Write, FileShare.None);
 
         await responseStream.CopyToAsync(fileTarget, 81920, cancellationToken);
```

Comment:
Now that we switched to netstandard2.1, we can omit to specify the buffer size (81920). This signature wasn't available on netstandard2.0